app/javascript/legacy/external_applications.js

```diff
@@ -1,17 +1,20 @@
 function setupExternalRedirectedListener() {
   console.log('setupExternalRedirectedListener');
   var changeElement = document.getElementById("external_app_redirection_data");
-  console.log(changeElement)
+  console.log(changeElement);
+
   if (changeElement != null) {
     var redirUrl = changeElement.getAttribute("data-redirect-url");
     var jwtVal = changeElement.getAttribute("data-redirect-jwt");
+
     if (redirUrl != null) {
       window.localStorage.setItem("jwt", jwtVal);
-      window.location.assign(redirUrl + "?auth_token=" + jwtVal);
+      window.localStorage.setItem("token", jwtVal);
```

Comment:
for some reason this line is flagged by codesweep but not the previous one, can we reuse the previous one ?